void main(List<String> args) {
  void currentUser(String mail, String userID) {
    try {
      checkUser() {}
    } on CustomException catch (e) {
      print(e);
    } finally {
      navigateToLogin(int errorCode) {}
    }
  }
}

class CustomException {}
